cmd/gcp-controller-manager/node_csr_approver_test.go

```diff
@@ -1189,6 +1189,7 @@ func TestShouldDeleteNode(t *testing.T) {
 					Name: "node-test",
 				},
 			},
+			instance: &compute.Instance{},
```

Comment:
@vinayakankugoyal Added an empty instance in this mock test case, without this instance, inst and err are both nil in unit test.